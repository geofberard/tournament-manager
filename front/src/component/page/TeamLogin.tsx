import { Grid, SxProps } from "@mui/material";
import * as React from "react";
import { FC } from "react";
import { LoginMenu } from "../view/LoginMenu";
import { TeamSelection } from "../view/TeamSelection";

const fullScreen: SxProps = {
    height: "100vh",
}

export const TeamLogin: FC = () => {
    return (
        <>
            <Grid container
                spacing={0}
                direction={{ xs: 'column', md: "row" }}
                sx={fullScreen}>
                <LoginMenu />
                <TeamSelection />
            </Grid>
        </>
    );
};
