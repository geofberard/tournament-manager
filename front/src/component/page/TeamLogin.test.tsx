import * as React from 'react';
import { testSnapshot } from '../../test/TestUtils';
import { TeamLogin } from './TeamLogin';

testSnapshot(<TeamLogin />)