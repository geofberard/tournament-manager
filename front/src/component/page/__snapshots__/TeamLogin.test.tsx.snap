// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match last snapshot 1`] = `
<React.Fragment>
  <ForwardRef(Grid)
    container={true}
    direction={
      Object {
        "md": "row",
        "xs": "column",
      }
    }
    spacing={0}
    sx={
      Object {
        "height": "100vh",
      }
    }
  >
    <LoginMenu />
    <TeamSelection />
  </ForwardRef(Grid)>
</React.Fragment>
`;
